fluentdhec/lambda_function.py

```diff
@@ -32,19 +34,41 @@ def lambda_handler(event, context):
         send_to_hec(payload)
 
 
+def extract_time(message):
```

Comment:
Is the `extract_time` method specific for the k8s time stamps or is it a generic function for all time stamps? Should its name, and possibly it's location reflect that?

Reply:
Idea was a generic function